### ISSUE TITLE
The training results differ significantly when using model=yolo11 and model=path.

### ISSUE THREAD

**Issue by magic524** [ultralytics/ultralytics#16636]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

When running commands on the same dataset using `model=yolo11n` and `model=/root/.../yolo11.yaml` (with other scales commented out, leaving only n), the models show the same parameter count, but the AP50 differs by 10 points.
![1](https://github.com/user-attachments/assets/775d329f-321f-46ba-b9cd-0718dcd0958d)
![2](https://github.com/user-attachments/assets/2472575e-4b3e-4dda-a9db-cbdd78b2576a)
![3](https://github.com/user-attachments/assets/6ce56280-b053-48e7-b5bc-e12fa41d78fa)
![4](https://github.com/user-attachments/assets/217a1e61-cac6-46eb-9e94-2f15d55381b2)
![5](https://github.com/user-attachments/assets/d312bb64-56fd-44a8-a733-51bd8086b939)

### Additional

I trained for 200 epochs on NightDrone, ensuring only the model setting changed, and got completely different results. Which one is correct?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @magic524, thank you for bringing your query to Ultralytics 🚀! We recommend checking out our comprehensive [Docs](https://docs.ultralytics.com) where you'll find numerous [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples which may address many common questions.

Since it seems you are observing significant differences in training results, we would greatly appreciate a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us debug this issue 🐛.

For custom training queries, please include additional details such as specific dataset images, complete training logs, and ensure you're following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join our Ultralytics community wherever fits you best. For real-time chat, hop into our [Discord](https://ultralytics.com/discord) 🎧. For more detailed discussions, check out [Discourse](https://community.ultralytics.com) or our [Subreddit](https://reddit.com/r/ultralytics) for sharing insights.

## Upgrade

Ensure your setup is up-to-date by upgrading to the latest `ultralytics` package, including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml), in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can be executed in these up-to-date verified environments (all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing. CI tests ensure correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) across macOS, Windows, and Ubuntu.

This is an automated response, but rest assured, an Ultralytics engineer will assist you soon! 🎉

**Comment by glenn-jocher** [comment]
@magic524 the difference in results could be due to variations in hyperparameters or initial weights between the two models. Ensure both models are initialized with the same weights and hyperparameters for a fair comparison. If the issue persists, consider checking the training logs for any discrepancies.
